signalfx-org-metrics/metrics.yaml

```diff
@@ -1,3 +1,255 @@
+sf.org.apm.numSpansDroppedThrottle:
+  brief: The number of spans SignalFx receives beyond the allowed ingest volume.
+  description: 'The number of spans SignalFx receives beyond the allowed ingest volume.
+    SignalFx drops spans it receives after the ingestion volume limit is reached.
+
+    Dimension(s): `orgId`
+    
+    Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numSpansDroppedThrottle
+
+sf.org.apm.grossSpansReceived:
+  brief: The number of spans SignalFx receives before filtering and throttling.
+  description: 'The number of spans SignalFx receives before filtering or throttling.
+
+  Dimension(s): `orgId`
+
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.grossSpansReceived
+
+sf.org.apm.grossSpansReceivedByToken:
+  brief: The number of spans SignalFx receives for a specific access token
+    before filtering and throttling.
+  description: 'The number of spans SignalFx receives for a specific access token
+    before filtering or throttling.
+
+    Dimension(s): `orgId`, `tokenId`
+    
+    Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.grossSpansReceivedByToken
+
+sf.org.apm.numSpansReceived:
+  brief: The number of spans SignalFx receives after filtering and throttling.
```

Comment:
Maybe use "accepted" instead of "received"

Reply:
I worry about using accepted for gross because this is the pre-filtering/pre-throttling number... so i think this actually is "received" while the "num" are "accepted". Kind of wish we had used that nomenclature when naming these...

Reply:
This _was_ commenting on the "num" metric, actually.